packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.tsx

```diff
@@ -52,7 +52,7 @@ export function newInitialState(): TTraceTimeline {
     detailStates: new Map(),
     hoverIndentGuideIds: new Set(),
     shouldScrollToFirstUiFindMatch: false,
-    spanNameColumnWidth: 0.25,
+    spanNameColumnWidth: parseFloat(localStorage.getItem('spanNameColumnWidth') || '0.25'),
```

Comment:
prettier linter seems to be failing on this

Reply:
I think it's failing below on a missing semicolon. I amended the commit.

Reply:
It is now unhappy about `scripts/get-changelog.js`, which I haven't touched, and I'm not sure what to do with that.

Reply:
I pushed a fix to it https://github.com/jaegertracing/jaeger-ui/commit/ec17f7896d669b21c3b34c9c9fd6946becd9687b, rebasing